modules/SofaConstraint/GenericConstraintCorrection.cpp

```diff
@@ -36,78 +36,97 @@ namespace component
 namespace constraintset 
 {
 
+using helper::vector;
+using std::string;
```

Comment:
This will depend on which rules we use...but at least don't import std::string and sofa::helper::vector